LibreNMS/OS/Shared/Cisco.php

```diff
@@ -610,10 +610,17 @@ public function discoverStpInstances(?string $vlan = null): Collection
         $vlans = $this->getDevice()->vlans;
         $instances = new Collection;
 
+        //get Cisco stpxSpanningTreeType
+        $stpxSpanningTreeType = \SnmpQuery::cache()->enumStrings()->hideMib()->get('CISCO-STP-EXTENSIONS-MIB::stpxSpanningTreeType.0')->value();
```

Comment:
Don't cache something unless it is used in two places in one poller run.  Otherwise LibreNMS' RAM usage will explode :)